src/qibolab/components/channels.py

```diff
@@ -27,8 +27,16 @@
 
 
 class Channel(Model):
-    name: ChannelId
-    """Name of the channel."""
+    """Channel to communicate with the qubit."""
+
+    device: str = ""
+    """Name of the device."""
+    path: str = ""
+    """Physical port addresss within the device."""
+
+    @property
+    def port(self) -> int:
+        return int(self.path)
```

Comment:
I don't think this would work for all instruments, at least probably not for ZI, so we could even "downgrade" internally to QM. It could be useful for other instruments though, which could justify keeping it here.

Reply:
ZI will use the `.path`, and most of the others will also use `.path` directly. 

I would say that it makes a bit of sense, since `int` and `str` are the only general types we can accommodate for, using `str` to represent both. If any driver needs anything more structured, it will make more sense to write the custom (de)serialization (from)to `str` in the driver itself, and treat it as `str` in the channel.
We could follow this logic for `int` as well, but that's the minimally structured one, so we can even afford (it's just a compromise).